ui/src/components/Shipnoise/AvailableRecordings.tsx

```
@@ -0,0 +1,356 @@
+      .toLowerCase()
+      .replace(/\s+/g, "-")
+      .replace(/[^\w-]/g, "");
+    const url = `https://live.orcasound.net/listen/${acceptedLabel}`;
+    window.open(url, "_blank");
+  };
+
+  useEffect(() => {
+    const existingScripts = document.querySelectorAll(
+      'script[src*="chimpstatic"]',
+    );
+    existingScripts.forEach((s) => s.remove());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)